specs/latest/2.0/index.html

```diff
@@ -830,6 +830,58 @@ <h2 class="no-toc">Table of contents</h2>
 
 <!-- ======================================================================================================= -->
 
+    <h4>More binding points</h4>
+
+    <dl class="methods">
+      <dt class="idl-code">void bindBuffer(GLenum target, WebGLBuffer? buffer)
+          <span class="gl-spec">(<a href="http://www.khronos.org/registry/gles/specs/3.0/es_spec_3.0.3.pdf#nameddest=section-2.9.1">OpenGL ES 3.0.3 &sect;2.9.1</a>, <a class="nonnormative" href="http://www.khronos.org/opengles/sdk/docs/man3/html/glBindBuffer.xhtml">man page</a>)</span>
+      </dt>
+      <dd>
+          Binds the given WebGLBuffer object to the given binding point(target). <em>target</em> is given in the following table:
+          <table>
+              <tr><th>target</th></tr>
+              <tr><td>ARRAY_BUFFER</td></tr>
+              <tr><td>ELEMENT_ARRAY_BUFFER</td></tr>
+              <tr><td>PIXEL_PACK_BUFFER</td></tr>
+              <tr><td>PIXEL_UNPACK_BUFFER</td></tr>
+              <tr><td>TRANSFORM_FEEDBACK_BUFFER</td></tr>
+              <tr><td>UNIFORM_BUFFER</td></tr>
+          </table>
+          <p>If <em>target</em> is not in the table above, generates an <code>INVALID_ENUM</code> error.</p>
```

Comment:
This must refer to the section farther down in the spec "Buffer Object Binding", which defines restrictions, similarly to the bindBuffer description in the WebGL 1.0 spec.